core/blocktree/blocktree.go

```diff
@@ -20,10 +20,11 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/ChainSafe/gossamer/core/types"
+
```

Comment:
I think we're seeing the result of this: https://github.com/golang/go/issues/20818